Jenkinsfile.integration

```diff
@@ -199,7 +200,7 @@ pipeline {
         }
         cleanup {
             script {
-                    sh './run.sh teardown'
+                    sh '[ -f /tmp/$SOCOK8S_ENVNAME.needcleanup ] && ./run.sh teardown && rm /tmp/$SOCOK8S_ENVNAME.needcleanup'
```

Comment:
I think we cannot leverage those `TestFunctional` and `TestDocs` vars for this rigth? Like they re not available? :(

Reply:
Good question. I think this is very simple, and I would be happy to see our CI gods improving over this, to feel it less hacky.

Reply:
IIRC @jhesketh tried this and it didnt work? those vars and the `when` keyword are not part of the `post` workflow